[bug] pkg.main, pkg.module are incompatible, so require("sortablejs") behaves differently under Rollup/Webpack

In some environments like Node and Browserify, `require("sortablejs")` returns `Sortable`, but in others like Rollup and Webpack, `require("sortablejs")` returns a module `{ Sortable: Sortable, default: Sortable, … }`. This means a SortableJS dependent can’t be compatible with these different environments without contortions.

```console
$ npm i browserify rollup @rollup/plugin-commonjs @rollup/plugin-node-resolve sortablejs webpack webpack-cli
$ echo 'console.log(require("sortablejs"))' > src.js
$ node src.js
[Function: Sortable] {
  utils: { … },
  get: [Function (anonymous)],
  mount: [Function (anonymous)],
  create: [Function (anonymous)],
  version: '1.10.2'
}
$ npx browserify src.js > out.js
$ node out.js
[Function: Sortable] {
  utils: { … },
  get: [Function (anonymous)],
  mount: [Function (anonymous)],
  create: [Function (anonymous)],
  version: '1.10.2'
}
$ npx rollup src.js -p node-resolve -p commonjs -f cjs -o out.js
$ node out.js
[Object: null prototype] {
  default: [Function: Sortable] {
    utils: { … },
    get: [Function (anonymous)],
    mount: [Function (anonymous)],
    create: [Function (anonymous)],
    version: '1.10.2'
  },
  MultiDrag: [Function: MultiDragPlugin],
  Sortable: [Function: Sortable] {
    utils: { … },
    get: [Function (anonymous)],
    mount: [Function (anonymous)],
    create: [Function (anonymous)],
    version: '1.10.2'
  },
  Swap: [Function: SwapPlugin]
}
$ npx webpack -d
$ node dist/main.js
Object [Module] {
  MultiDrag: [Getter],
  Sortable: [Getter],
  Swap: [Getter],
  default: [Function: Sortable] {
    utils: { … },
    get: [Function (anonymous)],
    mount: [Function (anonymous)],
    create: [Function (anonymous)],
    version: '1.10.2'
  }
}
```

This is because Node and Browserify are using `pkg.main` (`./Sortable.js`), while Rollup and Webpack are using `pkg.module` (`modular/sortable.esm.js`), and the two files do not provide compatible interfaces.

This problem is [called out](https://github.com/rollup/rollup/blob/master/docs/999-big-list-of-options.md#outputexports) in the Rollup documentation:

> Note: There are some tools such as Babel, TypeScript, Webpack, and `@rollup/plugin-commonjs` that are capable of resolving a CommonJS `require(...)` call with an ES module. If you are generating CommonJS output that is meant to be interchangeable with ESM output for those tools, you should always use `named` export mode. The reason is that most of those tools will by default return the namespace of an ES module on `require` where the default export is the `.default` property.

Three potential solutions are:
* use `named` export mode, so `require("sortablejs").default` or `require("sortablejs").Sortable` would work everywhere; or
* add `Sortable.default = Sortable` and/or `Sortable.Sortable = Sortable` for compatibility; or
* remove `pkg.module`.

**Information**

Versions - Look in your `package.json` for this information:
sortablejs = ^1.10.2

Now with sortablejs@1.11.2-alpha.3 (did you mean to tag an alpha as latest?), `require("sortablejs")` doesn’t work at all in Node because `pkg.main == "dist/sortable.js"` is missing entirely.

```console
$ npm i sortablejs

+ sortablejs@1.11.2-alpha.3
added 1 package and audited 1 package in 0.542s
found 0 vulnerabilities

$ node -e 'require("sortablejs")'
internal/modules/cjs/loader.js:328
      throw err;
      ^

Error: Cannot find module '/tmp/test/node_modules/sortablejs/dist/sortable.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:320:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:533:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:879:27)
    at Function.Module._load (internal/modules/cjs/loader.js:742:27)
    at Module.require (internal/modules/cjs/loader.js:964:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at [eval]:1:1
    at Script.runInThisContext (vm.js:132:18)
    at Object.runInThisContext (vm.js:309:38)
    at Object.<anonymous> ([eval]-wrapper:10:26) {
  code: 'MODULE_NOT_FOUND',
  path: '/tmp/test/node_modules/sortablejs/package.json',
  requestPath: 'sortablejs'
}

$ ls node_modules/sortablejs/dist
sortable.umd.js  sortable.umd.js.map
```

--- a/lerna.json
+++ b/lerna.json
@@ -5,6 +5,6 @@
     "packages/plugins/*",
     "packages/utils"
   ],
-  "version": "1.11.2-alpha.4",
+  "version": "1.12.0",
   "npmClient": "yarn"
 }

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "@sortable/monorepo",
+  "name": "@sortablejs/monorepo",
   "private": true,
   "scripts": {
     "test:compat": "ts-node scripts/test-compat.ts",

--- a/packages/plugins/auto-scroll/package.json
+++ b/packages/plugins/auto-scroll/package.json
@@ -1,4 +1,4 @@
 {
-  "name": "@sortable/plugin-auto-scroll",
+  "name": "@sortablejs/plugin-auto-scroll",
   "private": true
 }

--- a/packages/plugins/multi-drag/package.json
+++ b/packages/plugins/multi-drag/package.json
@@ -1,4 +1,4 @@
 {
-  "name": "@sortable/plugin-multi-drag",
+  "name": "@sortablejs/plugin-multi-drag",
   "private": true
 }

--- a/packages/plugins/on-spill/package.json
+++ b/packages/plugins/on-spill/package.json
@@ -1,4 +1,4 @@
 {
-  "name": "@sortable/plugin-on-spill",
+  "name": "@sortablejs/plugin-on-spill",
   "private": true
 }

--- a/packages/plugins/swap/package.json
+++ b/packages/plugins/swap/package.json
@@ -1,4 +1,4 @@
 {
-  "name": "@sortable/plugin-swap",
+  "name": "@sortablejs/plugin-swap",
   "private": true
 }

--- a/packages/sortable/package.json
+++ b/packages/sortable/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "@sortable/sortable",
+  "name": "@sortablejs/sortable",
   "version": "1.12.0",
   "private": true
 }

--- a/packages/sortable/package.json
+++ b/packages/sortable/package.json
@@ -1,5 +1,5 @@
 {
   "name": "@sortable/sortable",
-  "version": "1.11.2-alpha.4",
+  "version": "1.12.0",
   "private": true
 }

--- a/packages/sortablejs/package.json
+++ b/packages/sortablejs/package.json
@@ -1,6 +1,6 @@
 {
   "name": "sortablejs",
-  "version": "1.11.2-alpha.4",
+  "version": "1.12.0",
   "description": "JavaScript library for reorderable drag-and-drop lists on modern browsers and touch devices",
   "main": "dist/sortable.umd.js",
   "module": "modular/sortable.complete.esm.js",

--- a/packages/sortablejs/package.json
+++ b/packages/sortablejs/package.json
@@ -3,7 +3,6 @@
   "version": "1.12.0",
   "description": "JavaScript library for reorderable drag-and-drop lists on modern browsers and touch devices",
   "main": "dist/sortable.umd.js",
-  "module": "modular/sortable.complete.esm.js",
   "scripts": {
     "build": "npm-run-all --parallel build:*",
     "build:umd": "microbundle -f umd -i src/umd.ts -o dist/sortable.umd.js --name Sortable",

--- a/packages/utils/package.json
+++ b/packages/utils/package.json
@@ -1,4 +1,4 @@
 {
-  "name": "@sortable/utils",
+  "name": "@sortablejs/utils",
   "private": true
 }

